src/resources.md

```diff
@@ -21,7 +21,10 @@ Objecttype op [GEMMA Online](https://www.gemmaonline.nl/index.php/Imztc_2.1/doc/
 | publicatietekst | De generieke tekst van de publicatie van BESLUITen van dit BESLUITTYPE | string | nee | C​R​U​D |
 | publicatietermijn | Het aantal dagen, gerekend vanaf de verzend- of publicatiedatum, dat BESLUITen van dit BESLUITTYPE gepubliceerd moeten blijven. | string | nee | C​R​U​D |
 | toelichting | Een eventuele toelichting op dit BESLUITTYPE. | string | nee | C​R​U​D |
-| informatieobjecttypes |  | array | nee | ~~C~~​R​~~U~~​~~D~~ |
+| informatieobjecttypes |  | array | ja | C​R​U​D |
+| beginGeldigheid | De datum waarop het is ontstaan. | string | ja | C​R​U​D |
+| eindeGeldigheid | De datum waarop het is opgeheven. | string | nee | C​R​U​D |
+| draft | Flag indicating the object is a draft | boolean | nee | ~~C~~​R​~~U~~​~~D~~ |
```

Comment:
Can you replace `draft` with `concept`, with the following help_text:
   > "Geeft aan of het object een concept betreft. Concepten zijn niet-definitieve versies en zouden niet gebruikt moeten worden buiten deze API."